services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -341,6 +339,18 @@ func processEsls(ctx context.Context, repo repository.Repository, dbHandler *db.
 	}
 }
 
+func measurePushFailures(ddMetrics statsd.ClientInterface, log *zap.SugaredLogger, failure bool) {
```

Comment:
`measurePushFailures` is not what this does anymore, remove `Failures`?

Reply:
done